src/shared/codelens/localLambdaRunner.ts

```diff
@@ -310,3 +321,235 @@ export class LocalLambdaRunner {
         }
     }
 }
+
+export const makeBuildDir = async (): Promise<string> => {
+    const buildDir = await makeTemporaryToolkitFolder()
+    ExtensionDisposableFiles.getInstance().addFolder(buildDir)
+
+    return buildDir
+}
+
+export async function makeInputTemplate(params: {
+    baseBuildDir: string,
+    codeDir: string,
+    documentUri: vscode.Uri
+    handlerName: string,
+    runtime: string,
+    workspaceUri: vscode.Uri,
+}): Promise<string> {
+    const inputTemplatePath: string = path.join(params.baseBuildDir, 'input', 'input-template.yaml')
+    ExtensionDisposableFiles.getInstance().addFolder(inputTemplatePath)
+
+    // Make function handler relative to baseDir
+    const handlerFileRelativePath = path.relative(
+        params.codeDir,
+        path.dirname(params.documentUri.fsPath) // localInvokeParams.document.uri.fsPath
```

Comment:
Remove commented code (throughout this file)

Reply:
✔